app/controllers/chatroom_controller.rb

```diff
@@ -1,5 +1,8 @@
 class ChatroomController < ApplicationController
+  before_action :require_user
+
   def index
+    @message = Message.new
```

Comment:
??

Reply:
Will remove the unnecessary message instance 😅